docs/source/install_linux.rst

```diff
@@ -1,25 +1,136 @@
-OpenFAST
-========
+Building OpenFAST
+=================
+
+OpenFAST uses the `CMake <https://cmake.org>`__ build system. 
+We recommend building OpenFAST using `Spack <https://spack.readthedocs.io/en/latest>`__. 
+However, we also provide some sample scripts in ``openfast/share`` if you choose to proceed without `Spack`.
+
+Dependencies
+------------
+
+OpenFAST has the following dependencies:
+
+- LAPACK libraries provided through the variable ``BLASLIB``
```

Comment:
Is this an OpenFAST specific modification? Standard CMake variable is not `BLASLIB`, I think that is Intel MKL variable.

Reply:
@gantech was this `BLASLIB` variable only used for your script `fast-build.sh`?

Reply:
I think I used it only for fast-build.sh and specifically to compile with mkl. I'm not sure if it was required. Is there another path forward?

```CC=mpicc CXX=mpic++ FC=gfortran cmake \
   -DCMAKE_INSTALL_PREFIX=$openfast_dir/install/ \
   -DCMAKE_BUILD_TYPE=RELEASE \
   -DBUILD_FAST_CPP_API=ON \
   -DYAML_ROOT:PATH=$yaml_install_dir \
   -DHDF5_USE_STATIC_LIBRARIES=ON \
   -DHDF5_ROOT:PATH=$hdf5_install_dir \
   -DLAPACK_LIBRARIES="$BLASLIB" \
   -DBLAS_LIBRARIES="$BLASLIB" \
   -DFPE_TRAP_ENABLED=OFF \
   $EXTRA_ARGS \
../ &> log.cmake```

Reply:
@gantech that script can use the BLASLIB variable, but I think @sayerhs comment is that the line in the instructions is not accurate since the cmake or spack build processes do not involve explicitly setting that BLASLIB variable.

Reply:
How about this update:
```
OpenFAST has the following dependencies:

- LAPACK libraries
- For the optional C++ API, `HDF5 <https://support.hdfgroup.org/HDF5/>`__ (provided by ``HDF5_ROOT``) and `yaml-cpp <https://github.com/jbeder/yaml-cpp>`__ (provided by ``YAML_ROOT``)
- For the optional testing framework, Python 3+
```

Reply:
Yup... I agree. What is the Cmake standard for Lapack libraries though? May be I should be setting `LAPACK_DIR=${BLASLIB}` in the sample install scripts.

Reply:
Intel's `BLASLIB` includes both `-L` flags and `-l`. My recommendation would be to clarify in the documentation that the users should set `BLAS_LIBRARIES` and `LAPACK_LIBRARIES` appropriately if the library isn't found in standard paths. And use `BLASLIB` as an example.

Reply:
ok... Can I submit a separate pull request reg. this then?

Reply:
Push to this branch/pull request (assuming Raf gives you write access) it is a simple doc fix anyway.

Reply:
@gantech you should be able to submit edits to this pull request. lmk if not